Armadillo/src/main/java/com/scribd/armadillo/ArmadilloPlayerChoreographer.kt

```diff
@@ -308,7 +308,10 @@ internal class ArmadilloPlayerChoreographer : ArmadilloPlayer {
                 PlaybackState.PLAYING -> controls.pause()
                 PlaybackState.PAUSED -> controls.play()
                 else -> {
-                    stateModifier.dispatch(ErrorAction(UnexpectedException("Unknown state: $playbackState")))
+                    stateModifier.dispatch(ErrorAction(
```

Comment:
Somewhat out of scope, but the other `PlaybackState`s such as `STATE_STOPPED` or `STATE_BUFFERING` might not want to throw an exception of the user merely pressed play/pause during those times (I'm assuming it gets prevented downstream, though). Wonder if it's better to just do nothing

Reply:
Yeah ill investigate some issues ; i've been ramping up on this project so im hoping that more precise error messaging will help me understand what's going on.